app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
@JohnONolan / @kevinansfield So on the signin form there's currently an issue where any error messages that get displayed get kind of squished in between the forms. This is due to the selection above that declares `.gh-signin .form-group` to have less of a margin-bottom than all other form-groups.

I played with two approaches to fixing the squished error message text. One was to remove the shorter margin-bottom so it would use the default larger one. The other was this change, which only expands when there is an error class applied to the form-group. This change could in theory break when an `.error` class is applied to the form-group, but there's no error message present; however, given the structure of the validation form group components this won't happen. Although it does look a little odd to have it expand and collapse when the error message disappears and reappears.

My question is what the preferable behavior is? Here are some screenshots of the different results.

Currently:
<img width="447" alt="screen shot 2016-07-20 at 10 56 20 am" src="https://cloud.githubusercontent.com/assets/5167581/16995272/a96174b4-4e68-11e6-90a5-8c44a286f522.png">

With the first option (making the margin-bottom larger for error and non-error states):
http://recordit.co/TCv9P5qjak

With the second option (making the margin-bottom larger when the form-group has an error state):
http://recordit.co/81AyZ7ImVd

Personally, I don't have an opinion either way, so any advice is appreciated 😄